.github/workflows/prepare_release.yml

```
@@ -0,0 +1,92 @@
+name: Prepare release
+
+env:
+  GIT_AUTHOR_EMAIL: "packages@datadoghq.com"
+  GIT_AUTHOR_NAME: "ci.datadog-api-spec"
+
+on:
+  workflow_dispatch:
+    inputs:
+      tag:
+        description: New version tag
+        required: true
+
+jobs:
+  prepare_release:
+    name: Create release PR
+    runs-on: ubuntu-latest
+    steps:
+      - name: Get GitHub App token
+        id: get_token
+        uses: tibdex/github-app-token@v1
+        with:
+          app_id: ${{ secrets.PIPELINE_GITHUB_APP_ID }}
+          private_key: ${{ secrets.PIPELINE_GITHUB_APP_PRIVATE_KEY }}
+      - uses: actions/checkout@v2
+
+      - name: Setup Git
+        run: |
+          git config user.name "${GIT_AUTHOR_NAME}"
+          git config user.email "${GIT_AUTHOR_EMAIL}"
+
+      - name: Bump version
+        run: |
+          git switch -c "release/${RELEASE_TAG}"
+          sed -i 's/".*"/"${RELEASE_TAG#v}"/' version.go
+          git push -f --set-upstream origin "release/${RELEASE_TAG}"
```

you might want to commit first
```suggestion
          git commit -a -m "Version bump ${RELEASE_TAG}"
          git push -f --set-upstream origin "release/${RELEASE_TAG}"
```